.github/workflows/code-linting.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - uses: actions/setup-node@v1
+      - uses: actions/setup-node@v2
```

Comment:
You still need to remove the `with: node-version:` lines to automatically use a recent version of npm.

Once you do that the markdownlint should kick in again.